css/style.css

```diff
@@ -1,3 +1,44 @@
+body{
+  background-color: skyblue;
+}
 h1{
   color: darkblue;
-}
\ No newline at end of file
+  font-size: 75px;
+  text-align: center;
+
+}
+h2{
+  color: darkblue;
+  font-size: 50px;
+  text-align: center;
+}
+p{
+  font-size: 25px;
+  text-align: center;
+}
+a{
+  display:flex;
+  font-size: 25px;
+  text-align: center;
+  justify-content:space-evenly;
+  text-decoration: none;
+  font-weight: bold;
+  }
+  a:hover{
```

Comment:
This and the stuff below should be back one - even with the a{ above

A little note - it's usual to have a space between the selector and the bracket and between the colon and the rule: 
`a {` instead of `a{` 
`justify-content: space-evenly;` instead of `justify-content:space-evenly;`

Reply:
Oh! But the `a:hover` is correct - you don't want a space in that spot.